src/internal/AsyncSubject.ts

```diff
@@ -10,7 +10,7 @@ export class AsyncSubject<T> extends Subject<T> {
   private hasNext: boolean = false;
   private hasCompleted: boolean = false;
 
-  /** @deprecated This is an internal implementation detail, do not use. */
+  /** @internal This is an internal implementation detail, do not use. */
```

Comment:
I feel like we moved from internal to deprecated for some reason related to closure compiler or the like... cc @jasonaden

Reply:
TypeScript --stripInternal will remove @internal properties from the .d.ts which can affect downstream typechecking.

https://www.typescriptlang.org/docs/handbook/compiler-options.html

Reply:
ha interesting. Didn't knew about that. I'll fix that. Thanks for elaborating